hw/ip/otp_ctrl/otp_ctrl.core

```diff
@@ -8,12 +8,13 @@ description: "OTP Controller"
 filesets:
   files_rtl:
     depend:
+      - lowrisc:ip:otp_ctrl_pkg
       - lowrisc:ip:tlul
       - lowrisc:prim:all
       - lowrisc:prim:ram_1p
       - lowrisc:prim:otp
       - lowrisc:prim:lfsr
-      - lowrisc:ip:otp_ctrl_pkg
+      - lowrisc:ip:pwrmgr_pkg
```

Comment:
ahh...you may want to change your local lint run to use `design level = top`? 
Because `pwrmgr` is templated :) Or we can just do the changes for that one later when otp is ready to integrate.  

Both are fine with me.

Reply:
let me add that to the lint target. it should technically not matter since the signal definition this depends on is static.

Reply:
added! let me know if this is incorrect...